src/page-multi-facility/ReferenceDataModal/SyncNoUserFacilities.tsx

```diff
@@ -0,0 +1,315 @@
+
+  const stateNames = Array.from(localeDataSource.keys()).filter(
+    (name) => name !== "US Total",
+  );
+
+  const handleNextButtonClick = async () => {
+    setActiveStep(SYNC_REFERENCE_FACILITIES_CARD);
+  };
+
+  return (
+    <CardContent>
+      <CardDescription>
+        Recidiviz provides a forecast of likely COVID-19 cases in your
+        facilities based on policies and populations.
+      </CardDescription>
+      <StateAndSystemForm>
+        <StateNameSelection
+          stateName={stateName}
+          stateNames={stateNames}
+          setStateName={setStateName}
+        />
+        <SystemTypeSelection
+          systemType={systemType}
+          setSystemType={setSystemType}
+        />
+      </StateAndSystemForm>
+      <CardNavigation>
+        <InputButton
+          styles={{
+            width: "80px",
+          }}
+          label="Next"
+          onClick={handleNextButtonClick}
+        />
+      </CardNavigation>
+    </CardContent>
+  );
+};
+
+type SyncReferenceFacilitiesCardProps = CardStateProps & {
+  stateName: StateNameProps["stateName"];
+  systemType: SystemTypeProps["systemType"];
+};
+
+const SyncReferenceFacilitiesCard: React.FC<SyncReferenceFacilitiesCardProps> = (
+  props,
+) => {
+  const { stateName, systemType, setActiveStep, setModalTitle } = props;
+
+  const [scenarioState, dispatchScenarioUpdate] = useScenario();
+  const scenario = scenarioState.data;
+  const {
+    actions: { createOrUpdateFacility, fetchReferenceFacilities },
+  } = useFacilities();
+  const [referenceFacilities, setReferenceFacilities] = useState<
+    ReferenceFacility[]
+  >([]);
+  const [selectedFacilities, setSelectedFacilities] = useState<
+    ReferenceFacility[]
+  >([]);
+
+  useEffect(() => {
+    setModalTitle("Prepoulate Data");
+
+    const retrieveReferenceFacilities = async () => {
+      if (!stateName || !systemType) return;
+
+      const referenceFacilityMapping = await fetchReferenceFacilities(
+        stateName,
+        systemType,
+      );
+      const facilities = Object.values(referenceFacilityMapping);
+
+      setReferenceFacilities(facilities);
+      // By default, select all of the available Reference Facilities
+      setSelectedFacilities(facilities);
+    };
+    retrieveReferenceFacilities();
+  }, []);
+
+  const handleSelection = (refFacility: ReferenceFacility) => {
+    const selectedFacilitiesCopy = [...selectedFacilities];
+
+    if (selectedFacilitiesCopy.includes(refFacility)) {
+      let index = selectedFacilitiesCopy.indexOf(refFacility);
+      selectedFacilitiesCopy.splice(index, 1);
+    } else {
+      selectedFacilitiesCopy.push(refFacility);
+    }
+
+    setSelectedFacilities(selectedFacilitiesCopy);
+  };
+
+  const handleBackButtonClick = () => {
+    setActiveStep(STATE_AND_SYSTEM_SELECTION_CARD);
+  };
+
+  const handleSave = async () => {
+    const minimalModelInput = {
+      observedAt: new Date(),
+      updatedAt: new Date(),
+    };
```

Comment:
I think you probably also want to include `stateName` and (if present) `countyName` here?